.github/workflows/nuget-publish.yml

```diff
@@ -0,0 +1,70 @@
+name: NuGet Publish
+
+on:
+  push:
+    branches: [ master ]
+
+env:
+  NUGET_DIR: ${{ github.workspace }}/nuget
+  NUGET_ARTIFACT: nuget
+
+jobs:
+  testing:
+    if: contains(github.event.head_commit.message, 'chore(release)') 
+    uses: ./.github/workflows/test.yml
+
+  pack: 
+    if: contains(github.event.head_commit.message, 'chore(release)') 
+    runs-on: ubuntu-latest
```

Comment:
Let's pin this and all other `ubuntu`s to the specific version, similar to https://github.com/ButterCMS/buttercms-ruby/pull/28/files#diff-2c84033033d49186c63e6adcd705f63b11ae6814cd76c152c9c486d389fbccf3R23, to avoid unexpected failure in CI/CD when new Ubuntu is released with breaking changes.

Reply:
Resolved in https://github.com/ButterCMS/buttercms-csharp/pull/40/commits/f3d2d5e3f00c7bfcdd3405cd42a7cfb56b556a32